vignettes/articles/modify-functions.Rmd

```diff
@@ -0,0 +1,197 @@
+
+```{r tbl}
+x <- glm(response ~ age + grade, trial, family = binomial()) |>
+  tbl_regression(exponentiate = TRUE)
+x
+```
+
+## Column Formatting Modifiers
+
+- `modify_column_alignment()`: Update column alignment/justification
+- `modify_column_indent()`: Update column indentation
+- `modify_column_hide()`/`modify_column_unhide()`: Hide/unhide table columns
+
+These functions are used to apply column-wise formatting updates to a table.
+
+For each of these functions the column(s) on which to operate must be specified via the `columns` argument. 
+
+In addition to specifying the column(s) to modify, the `modify_column_indent()` function also optionally accepts a `rows` argument if indentation should only be changed on a particular subset of rows.
+
+```{r col fmt}
+x |>
+  # unhide the standard error column
+  modify_column_unhide(columns = std.error) |>
+  # align the p-value column to the right
+  modify_column_alignment(columns = p.value, align = "right") |>
+  # increase indentation to 8 for all labels from non-label rows
+  modify_column_indent(columns = label, rows = !row_type %in% 'label', indent = 8L)
+```
+
+## Cell Style Modifiers
+
+- `modify_bold()`/`remove_bold()`: Add/remove bold styling in table cells
+- `modify_italic()`/`remove_italic()`: Add/remove italic styling in table cells
+
+These functions are used to modify the cell font styling within a table.
+
+For each of the `modify_*()` functions the column(s) and row(s) in which cell contents should be bolded/italicized must be specified via the `columns` and `rows` arguments.
+
+The `remove_*()` functions will remove all bold/italic styling by default, but cell locations can be specified if needed via the `columns` and `rows` arguments.
+
+```{r cell style}
+x |>
+  # bold contents of label column in label rows
+  modify_bold(columns = label, rows = row_type == "label") |>
+  # italicize contents of p-value column in level rows
+  modify_italic(columns = p.value, rows = row_type == "level")
+```
+
+## Value Formatting Modifiers
+
+- `modify_missing_symbol()`: Update how missing values are represented in the table
+- `modify_fmt_fun()`: Update the formatting functions applied to numeric columns and rows
+- `modify_post_fmt_fun()`: Apply a second formatting function after the primary formatting functions have been applied
+
+These functions are used to modify the value formatting within a table.
+
+The formatting function modifiers can update multiple formats at once by listing the formats that should be applied in each column, as shown in the following example. The function can be called consecutively if different row subsets should be updated in each column.
+
+The `modify_post_fmt_fun()` function applies formatting to already formatted cell contents, which allows for conditions contingent on the text in the cell.
+
+```{r val fmt}
+x |>
+  # update missing value symbol from '-' to '<Missing>' for OR/CI columns in the reference group row
```

Comment:
To make this more realistic, perhaps it should be changed to `'Ref.'` for reference group?

Reply:
Not relevant in updated example